src/forklift/messaging.py

```diff
@@ -9,43 +9,62 @@
 import logging
 import secrets
 from config import get_config_prop
+from email import encoders
+from email import generator
+from email.mime.base import MIMEBase
 from email.mime.text import MIMEText
 from email.mime.multipart import MIMEMultipart
+from os.path import isfile
+from os.path import join
 from smtplib import SMTP
 
 
 log = logging.getLogger('forklift')
 
 
-def send_email(to, subject, body):
+def send_email(to, subject, body, attachment, save_to_file=False):
```

Comment:
We use this method in other places and I don't think that we want to change the required parameters. Can we give `attachment` a default value?